kubernetes/cache/proxy_status.go

```diff
@@ -41,7 +40,7 @@ func (c *kialiCacheImpl) pollIstiodForProxyStatus(ctx context.Context) {
 					interval := c.tokenNamespaceDuration / 2
 					retryErr := wait.PollImmediateUntilWithContext(ctx, interval, func(ctx context.Context) (bool, error) {
 						log.Trace("Getting proxy status from istiod")
-						proxyStatus, err = c.istioClient.GetProxyStatus()
+						proxyStatus, err = c.clientFactory.GetSAHomeClusterClient().GetProxyStatus()
```

Comment:
Just a thought, in some other places I read that eventually we will need to treat all clusters equally with no home cluster distinction but in some other places I see this method, I think in some way, we will need to define that one of them is the "home", right?. Not sure why yet but sounds logical.

Reply:
Yeah there's a couple different services that do autodiscovery that will need to know the "home" cluster at least in addition to this case where you want to use the client of the cluster kiali is deployed in to do the polling of istiod.